docs/setup-guide.md

```diff
@@ -1,27 +1,37 @@
 # Project Setup
+
 This file describes the setup process of the repository, with the commands and steps needed to succesffuly recreate it
 
 ## General information
+
 The repository is part of a Treecreate organization.\
 It utilizes Github Project, Issues and pull requests for task management.\
 Commits are made using [git-cz](https://github.com/streamich/git-cz), which replaces `git commit` and provides extra structure for making commits.\
 This is used in combination with [semantic-release](https://github.com/semantic-release/semantic-release), which handles automatic creation of releasses
 To use **git cz**, you have to install it globally with
+
 ```shell
-npm install -g commitizen git-cz
+npm install -g commitizen
+npm install -g git-cz
 commitizen init git-cz --save-dev --save-exact
 ```
+
 If you install it locally, you will have to use `npx git cz` instead
 
+For managing the monorepo, we use [NX](https://github.com/nrwl/nx).\
+Install it with `npm install -g nx` _(or install it locally and use it with `npx nx ...`)_
+
 ## Workspace setup
-Create a new workspace in the root folder of the repository
-`npx create-nx-workspace@latest webstore`
+
+Create a new workspace within the root folder of the repository
+Run `npx create-nx-workspace@latest webstore` from outside of the repository root directory _(it will create the workspace within the webstore directory)_
```

Comment:
I think it would be useful to mention that this command will not work with the current "latest" version of NX and that one of the currently known stable versions ( such as 11.1.1 or 11.1.5 ) can be used.

Reply:
Addresses with 3ca1c0c
This issue is also fixed by updating Node and NX to latest stable versions.